Scheduler/observing_blocks_maintel/AOS/BLOCK-T81.json

```diff
@@ -0,0 +1,594 @@
+                },
+                "rot_value": -68.4,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -61.2,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -54.0,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -46.8,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -39.6,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
```

Comment:
The same for n_triplets, the default is 1, so we probably don't need this here

Reply:
Resolved